cmd/internal/cli/actions.go

```diff
@@ -225,6 +219,10 @@ var ExecCmd = &cobra.Command{
 	PreRun:                replaceURIWithImage,
 	Run: func(cmd *cobra.Command, args []string) {
 		a := append([]string{"/.singularity.d/actions/exec"}, args[1:]...)
+		if wantsVM(cmd) {
```

Comment:
Let's move this into its own function. As noted previously, just create `func ActionRun` and then here do: 

```
Run: ActionRun
```

Same comment applies for all of these related commands

Reply:
Actually, this is kinda dumb, I don't think that makes sense here.